plugins/org.jkiss.dbeaver.ext.altibase/src/org/jkiss/dbeaver/ext/altibase/AltibaseUtils.java

```diff
@@ -72,4 +72,16 @@ public static String getEffectiveSql(String script) {
         
         return ddl.substring(0, last);
     }
+    
+    public static String getDottedName(String schemaName, String tableName, String partName) {
+        return getDottedName(new String[] {schemaName, tableName, partName});
+    }
+    
+    public static String getDottedName(String[] nameParts) {
+        int i = 0;
+        return new StringBuilder().append(nameParts[i++])
+                .append(".").append(nameParts[i++])
+                .append(CommonUtils.isEmpty(nameParts[i]) ? "" : "." + nameParts[i])
+                .toString();
+    }
```

Comment:
I suggest instead using `DBUtils#getFullyQualifiedName`

Reply:
> I suggest instead using `DBUtils#getFullyQualifiedName` or `DBUtils#getSimpleQualifiedName`

Thanks, it's looks better :)